package.json

```diff
@@ -49,9 +49,14 @@
     "crypto-js": "3.1.9-1",
     "get-value": "^3.0.1",
     "is": "^3.3.0",
-    "lodash": "^4.17.21",
     "lodash.clonedeep": "^4.5.0",
```

Comment:
Specific packages were already used previously, but at some point the global `lodash` package was added, which pulls in every single lodash function 😱  a lot of which aren't needed.

Reply:
Yes, it has been addressed for the core SDK, but integrations were left out.